### DISCUSSION TITLE
Clarifications on Final Project Submission for Hackathon

### DISCUSSION THREAD

**Discussion by tocar81** [msusdev/Innovation-Challenge#7] [Q&A]
Hi,

Our team has some questions about the final project submission for the hackathon:

1. Will we need to present our solution live to a panel of judges, or is it sufficient to meet the requirements listed on the hackathon About page?

2. Is there a specific or recommended maximum duration for the presentation video?

3. Are there specific requirements regarding the length of the PowerPoint presentation?

Thanks in advance.

**Comment by MatthewCalder-msft** [comment]
The primary thing is the video of you explaining your project and doing a demo if you can. Try to keep the video to about 10-15 minutes. For the powerpoint presentation, there are no specific requirements. Brief and focused is always good, but it is also an opportunity to present additional information, so it can be as long as you'd like.

**Reply by demersonpolli** [reply]
I have a question about the deadline for the Innovation Challenge December 2024. In the official rules is said December 21st, 2024 is the end of the entry phase. I have understand it was the deadline for submitting the application POC (*proof-of-concept*). When is the deadline for submission and, what is needed to be submitted? Only a presentation of the idea or a working application? Thanks.

**Reply by MatthewCalder-msft** [reply]
The official rules state the contest ends at 11:59 Pacific December 20. (UTC -8) You need to submit a video presentation, including a demo of your project. You also need to include a link to a presentation about your project (same one you use in your video) and a link to your a repo for your project. 
If you don't get as far as building a working application, you should still submit as much as you can to show the judges your ideas and how you came up with them. You can still get points even without a working POC app.

**Comment by demersonpolli** [comment]
Thank you for the clarification. It was very helpful.

Polli

**Reply by MatthewCalder-msft** [reply]
This might also help (how to upload your video) [https://github.com/msusdev/Innovation-Challenge/discussions/9](https://github.com/msusdev/Innovation-Challenge/discussions/9)

**Comment by demersonpolli** [comment]
Thank you!
